src/Controls/src/Core/Handlers/Items/CarouselViewHandler.Android.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Android.Views;
+using AndroidX.RecyclerView.Widget;
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Controls.Handlers.Items
+{
+	public partial class CarouselViewHandler : ItemsViewHandler<CarouselView>
+	{
+		double _widthConstraint;
+		double _heightConstraint;
+
+		protected override IItemsLayout GetItemsLayout() => VirtualView.ItemsLayout;
+
+		protected override ItemsViewAdapter<CarouselView, IItemsViewSource> CreateAdapter()
+		{
+			return new CarouselViewAdapter<CarouselView, IItemsViewSource>(VirtualView, (view, context) => new SizedItemContentView(Context, GetItemWidth, GetItemHeight));
+		}
+		protected override RecyclerView CreateNativeView()
+		{
+			return new MauiCarouselRecyclerView(Context, GetItemsLayout, CreateAdapter);
+		}
+
+		public static void MapIsSwipeEnabled(CarouselViewHandler handler, CarouselView carouselView)
+		{
+			(handler.NativeView as IMauiCarouselRecyclerView).IsSwipeEnabled = carouselView.IsSwipeEnabled;
+		}
+
+		public static void MapIsBounceEnabled(CarouselViewHandler handler, CarouselView carouselView)
+		{
+			handler.NativeView.OverScrollMode = carouselView?.IsBounceEnabled == true ? OverScrollMode.Always : OverScrollMode.Never;
```

Comment:
Could we have some basic Device tests validating the basic properties?. For example, this one is really easy to validate (get the native OverScrollMode property value).